### DISCUSSION TITLE
Tree Entities 可以查询出来tree，但是这结果是无序的，应该加一个排序的参数

### DISCUSSION THREAD

**Discussion by renjiali** [midwayjs/midway#3802] [Q&A]
### Describe the problem（描述问题）

Tree Entities 可以查询出来tree，但是这结果是无序的，应该加一个排序的参数

### Midway Versions（Midway 版本）

Tree Entities 可以查询出来tree，但是这结果是无序的，应该加一个排序的参数

**Comment by czy88840616** [comment]
应该提给 typeorm ?
